me.marzeq.xbox-cloud-gaming-electron.yaml

```diff
@@ -0,0 +1,58 @@
+app-id: me.marzeq.xbox-cloud-gaming-electron
```

Comment:
There is no `marzeq.me` domain found. This mean the app-id does not meet the requirements.

Reply:
That's my old domain, I sometimes forget I have a new one. Will update to my current one